insights/parsers/satellite_mongodb_storage_engine.py

```diff
@@ -44,7 +44,10 @@ def parse_content(self, content):
         self.name = ''
         for line in content:
             if line.strip().startswith('"name"'):
-                self.name = line.split(':')[1].strip(',').strip().strip('"')
-                break
+                try:
```

Comment:
The following logic makes the usage of this parser very limit, since except for the "name", all other information is discarded.  For a parser, we'd better make is as useful as we can, I mean, for this one, we'd better parser all the items inner the "{}".

Reply:
> discarded

Yes, I thought about it. But I don't think our rules will use all the other info. So I just parse the name out. What's your opinion? Do you think we'll use them?

Reply:
It depends. If the workload to parse extra info is heavy but it's obvious the extra info is useless, then we'd better skip them. But in this case, I don't see any difficulty to parse the reset lines other than the "name".  And,  it doesn't make sense that the rule you are working on doesn't use other lines indicate no rules will use them, right?